### ISSUE TITLE
Ignore suggest_dimension parameter in rule D101

### ISSUE THREAD

**Issue by emorrell23** [spectacles-ci/lkmlstyle#21]
Rule D101 (Dimension name not in snake case) is flagging a dimension for a field name in the suggest_dimension parameter of a dimension, not the dimension name itself. I believe it is flagging it because of the "." separating the `view_name` and `field_name`, but this format is required by Looker when using in combination with the suggest_explore parameter. Can we ignore the suggest_dimension parameter for this rule? 

```
[D101] Dimension name not in snake case
.../views/merchant_dim.view.lkml:49
───────────────────────────────────────────────────────────
47    | dimension: merchant_name { 
48    |     suggest_explore: suggest_merchant_name
 49   >|     suggest_dimension: merchant_dim.merchant_name_suggest_only
 50    |     type: string
 51    |     sql: ${TABLE}."MERCHANT_NAME" ;;
```

**Comment by joshtemple** [comment]
This fix is out in version 0.0.9. This rule will no longer check `suggest_dimension`. Thanks for reporting!
